src/TicketMap/index.tsx

```diff
@@ -336,8 +336,13 @@ export default class TicketMap extends Component<Props & DefaultProps, State> {
       return Object.keys($ticketGroupsBySectionByZone(this.state)).forEach(zone =>
         this.fillZone(zone.toLowerCase(), $areAllSectionsInTheZoneSelected(this.state)(zone.toLowerCase())))
     }
-    Object.keys($ticketGroupsBySection(this.state)).forEach(section =>
-      this.fillSection(section.toLowerCase(), this.state.selectedSections.has(section)))
+    Object.keys($ticketGroupsBySection(this.state)).forEach(section => {
+      this.fillSection(section.toLowerCase(), this.state.selectedSections.has(section))
+
+      for (const path of this.getAllPaths(section)) {
```

Comment:
Rather than looping over `this.getAllPaths(section)` (we're already doing this in `fillPathsForSection` which is called by `fillSection`, how about adding this as a property to be set in `fillSection`?
```javascript
fillSection (section: string, shouldHighlight = true) {
    const isAnAvailableSection = $venueSections(this.state).includes(section)
    if (isAnAvailableSection) {
      this.fillPathsForSection(() => ({
        'fill': this.getDefaultColor($ticketGroupsBySection(this.state)[section]),
        'opacity': shouldHighlight ? '1' : '0.6',
        'stroke-width': '1',
        'stroke': shouldHighlight ? '#4a4a4a' : '#FFFFFF',
        'cursor': shouldHighlight ? 'pointer' : 'default'
      }), section)
    }
  }
```

Reply:
I was trying to make sure the cursor style was only updated when ticketGroups changed. I noticed `fillSection` is called by the mouse-over and mouse-leave handlers, whereas it seemed that `updateMap` was only called as a side-effect of updating the ticket groups. Upon closer inspection, it looks like `updateMap` is also eventually called by the mouse-out handler, so I'll move this style change to `fillSection`. Nice catch!